app/views/agent/partials/vatReturnsPartial.scala.html

```diff
@@ -0,0 +1,38 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import common.MandationStatus
+@import java.time.Year
+
+@(mandationStatus: String)(implicit messages: Messages,appConfig: AppConfig){
+<div class="card" id="cardAmount">
```

Comment:
We try and avoid putting `id` in places that don't need it for any CSS styling specific to the element. You can use other CSS selectors in your unit tests to find that element (see comment on partial spec). Please review this file and remove other `id` attributes